newscoop/install/Resources/sql/upgrade/4.4.x/2015.03.11/topics.php

```diff
@@ -62,6 +62,14 @@
 });
 
 try {
+    $app['db']->query('ALTER TABLE `topic_translations` ADD `isDefault` INT( 1 ) NULL DEFAULT NULL');
+} catch (\Exception $e) {
+    if ($app['db']->errorCode() !== '42000') {
+        $logger->addWarning($e->getMessage());
+    }
+}
+
+try {
     // add TreeListener and TranslatableListener to EventManager,
```

Comment:
If you update an old upgrade file it could cause issues. Can you copy the query to a new folder with the name of todays date?